### PR TITLE
update trysmalltalk to amber 0.14.3

### LINKED ISSUE #24: Update amber-website to 0.14.3
Planned activity
1. fork repository https://github.com/amber-smalltalk/amber-website

```
git clone https://github.com/hhzl/amber-website
cd amber-website
git checkout -b gh-24
npm update amber-dev
bower install
git add -A
git commit -m "update to 0.14.3"
git push origin gh-24
```

Is this all that is needed?

### PR DESCRIPTION
**Pull request by tomrake**
Merge of this PR will change the LIVE  amber-lang.org website!!! See https://tomrake.github.io/ to preview what these changes.

Things I note 
1) Try Smalltalk in your browser will include the TrySmalltalk classes in the window it opens
2) Added a version to learn.html based on methods of https://github.com/amber-smalltalk/amber-website/pull/22

Please review and see if these changes are acceptable for the live site.

This should close #24

Most of the changes in the PR are dependencies in bower_components.

### CODE REVIEW

**Comment by ghost** [comment]
Two small issues:
- will you remove node_modules from repo and add it to .gitignore (that's maybe different issue)?
- why use `#commit-hash` in git url? Wouldn't it be better to add release tags in trysmalltalk directly and end it with `#^0.1.1` instead?

**Comment by tomrake** [comment]
On node_modules, I made the assumption that because I used `npm install` to create the gh-pages branch and that branch is essentially served by github for amber-lang.org that the node_modules are necessary and should not be in .gitignore. I will test removal at https://tomrake.github.io/ and see if I get an error.

On version tags
There are no version tags for amber-smalltalk/trysmalltalk I agree version tags, would be best.

**Comment by hhzl** [comment]
Minor thing: https://tomrake.github.io/learn.html
should have

```
 copyright 2011-2015
```

note.

**Comment by ghost** [comment]
You only need npm modules to run the tooling, on your machine. To run in browser, amber only needs things in `bower_components` when in devel mode, and even those can be eschewed in deploy mode, as there you only need the .html and the.js.

**Comment by hhzl** [comment]
Tom, thank you for working on this.

Preferrably we only should have the deployment version of 'the.js' and
'index.html' in the gh-pages branch.

No npm modules, no bower_components.

But I do not mind if for 0.14.3 we still have bower_components to not
further delay the announcment of 0.14.3.

**Comment by tomrake** [comment]
node_modules are not needed to run the site. I added a README.md mainly to remind me what to do the next time.  Once again see it at https://tomrake.github.io/ before making it go live on the site.

**Comment by hhzl** [comment]
Good to have the addition in the README.

File: bower.json [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+    "Amber",
+    "Smalltalk"
+  ],
+  "license": [
+    "MIT"
+  ],
+  "private": false,
+  "dependencies": {
+    "amber": "0.14.3",
+    "trysmalltalk": "0.1.1"
```
Comment (ghost) [inline bower.json:30]: Why only "0.1.1"? What's the reason to deny subsequent versions?

Comment (ghost) [inline bower.json:30]: IOW, do not update bower.json by hand, use `bower install` do it for you. Had you do the simple `bower install trysmalltalk --save`, bower would include other versions as well. IMO it is a bad habit to get used to edit bower.json manually and then running `bower install` (same for npm).

File: learn.html [hunk 2]
```diff
@@ -33,7 +35,7 @@
     
     <div id="footer">
       <div class="main">
-        <p>Copyright &copy; 2011 <a href="http://www.nicolas-petton.fr/">Nicolas Petton</a>. The content of this website in licensed under <a href="http://creativecommons.org/licenses/by-nc/3.0/">CC-BY-NC 3.0</a>.</p>
+        <p>Copyright &copy; 2011-2015 <a href="http://www.nicolas-petton.fr/">Nicolas Petton</a>. The content of this website in licensed under <a href="http://creativecommons.org/licenses/by-nc/3.0/">CC-BY-NC 3.0</a>.</p>
```
Comment (ghost) [inline learn.html:38]: AFAIK the list of years in copyright notice should only include years when substantial (that is, copyrightable) change was made. Maybe `2011, 2015` is more appropriate, then. Just my feeling, though.

**Comment by hhzl** [comment]
In 2014 we had the move to the new `amber init` type of project style. Quite a substantial change.

**Comment by ghost** [comment]
Was it in 2014 already (I mean the website; I hope I am not totally off and the copyright notice is for the website (it's a question for what it is, though, hm...))? Than yes, I would write `2011, 2014-2015`. And if it is for Amber itself, then of course 2011-2015 is correct.

**Comment by ghost** [comment]
Ah, I see, this is trysmalltalk demo page. So it is for that.

**Pull request merged**